{"status": "failed", "attachments": [{"source": "029af138-61aa-4101-96f9-2a54dc780d71-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "c91c8679-296c-471c-a9be-1a975ff4ac52-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "8a618df4-e93a-41e4-838b-9c9b994e8ec0-attachment.txt", "type": "text/plain", "name": "stderr"}], "labels": [{"name": "story", "value": "Platform-fsp_sss-1711:NC跨国不走专线收发音视频，共享媒体文件，共享桌面声音"}, {"name": "feature", "value": "必测用例"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_checklist"}, {"name": "subSuite", "value": "Test_CheckList"}, {"name": "host", "value": "DESKTOP-JPRBBOB"}, {"name": "thread", "value": "28496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "TestCase.test_checklist"}], "stop": 1542803385648, "uuid": "a2b34408-942e-45de-9e97-14fa653b3eba", "historyId": "c4f754fe91149f6afa63667a9e830ad0", "start": 1542803328668, "steps": [{"status": "failed", "start": 1542803330676, "stop": 1542803385647, "name": "验证音视频功能", "statusDetails": {"message": "AssertionError: assert False\n", "trace": "  File \"D:\\pyprojects\\test_fsp\\TestCase\\test_checklist.py\", line 533, in test_nc_transnational_not_specialline\n    assert result\n"}}], "fullName": "TestCase.test_checklist.Test_CheckList#test_nc_transnational_not_specialline", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <TestCase.test_checklist.Test_CheckList object at 0x05162CB0>\nsftp = <paramiko.sftp_client.SFTPClient object at 0x05043050>\n\n    @allure.story('Platform-fsp_sss-1711:NC跨国不走专线收发音视频，共享媒体文件，共享桌面声音')\n    def test_nc_transnational_not_specialline(self, sftp):\n    \n        base.use_specialline(False)\n        self.changeRuleConf_step(sftp, 'rule-config_Transnational.xml')\n        with allure.step('验证音视频功能'):\n            send_flag, recv_flag, result = check_nc_av()\n>           assert result\nE           assert False\n\nTestCase\\test_checklist.py:533: AssertionError"}, "name": "test_nc_transnational_not_specialline"}